lib/features/composer/presentation/composer_controller.dart

```diff
@@ -141,9 +141,6 @@ class ComposerController extends BaseController {
       return newContentHtml;
     } else {
       String contentHtml = await htmlEditorApi?.getText() ?? '';
-      if(Platform.isAndroid) {
-        contentHtml = contentHtml.replaceAll('&quot;', '"');
-      }
```

Comment:
I did not find the replacement for it? Why we don't need it at all?

Reply:
In the new code fork of the original library, they didn't do `replace` the special characters when `getText` so we won't need to `replace` these characters again.